backend/src/test/java/gov/cdc/usds/simplereport/db/repository/BaseRepositoryTest.java

```diff
@@ -15,14 +15,20 @@
 import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
 import org.springframework.context.annotation.Import;
 import org.springframework.test.context.ActiveProfiles;
+import org.thymeleaf.spring6.SpringTemplateEngine;
 
 /**
  * A base test for Spring Data repository tests. Comes pre-wired with a standard API user to attach
  * to JPA audit events: does not support patient-specific operations.
  */
 @DataJpaTest
 @AutoConfigureTestDatabase(replace = Replace.NONE)
-@Import({SliceTestConfiguration.class, DbTruncator.class, DataSourceConfiguration.class})
+@Import({
+  SliceTestConfiguration.class,
+  DbTruncator.class,
+  DataSourceConfiguration.class,
+  SpringTemplateEngine.class
```

Comment:
⚠️ If I imported `SpringTemplateEngine.class` in the `SliceTestConfiguration.java` class file then the service tests would fail because of a duplicate bean issue so I'm only importing it here to make repository tests pass